### PR TITLE
[MIG] stock_picking_send_by_mail: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@cristinamartinrod please review

File: stock_picking_send_by_mail/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,441 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (chienandalu) [inline stock_picking_send_by_mail/static/description/index.html:1]: Should this be removed?

File: stock_picking_send_by_mail/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 - Sandra Figueroa Varela
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking by Mail",
+    "summary": "Send stock picking by email",
+    "version": "11.0.1.0.0",
```
Comment (bodedra) [inline stock_picking_send_by_mail/__manifest__.py:8]: Can you add following key:

    "website": "https://github.com/OCA/stock-logistics-workflow",

File: stock_picking_send_by_mail/models/stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            'mail.email_compose_message_wizard_form',
+            False,
+        )
+        ctx = dict(
+            default_model='stock.picking',
+            default_res_id=self.id,
+            default_use_template=bool(template),
+            default_template_id=template and template.id or False,
+            default_composition_mode='comment',
+            mark_invoice_as_sent=True,
```
Comment (bodedra) [inline stock_picking_send_by_mail/models/stock_picking.py]: I think no need of `mark_invoice_as_sent=True` so we can remove that.

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:  A minor comment below:

File: stock_picking_send_by_mail/static/description/index.html (hunk 1 above)
Comment (cubells) [reply stock_picking_send_by_mail/static/description/index.html:1]: @chienandalu this file is created automatically by oca-gen-readme script.

Comment (pedrobaeza) [reply stock_picking_send_by_mail/static/description/index.html:1]: It's created automatically by README generator, so I guess no.

Comment (chienandalu) [reply stock_picking_send_by_mail/static/description/index.html:1]: Ok, I didn't noticed about it before

File: stock_picking_send_by_mail/__manifest__.py (hunk 2 above)
Comment (cubells) [reply stock_picking_send_by_mail/__manifest__.py:8]: @bodedra done.

**Comment by pedrobaeza** [comment]
As it has already 3 reviews, please squash last 2 commits for merging after CI.

**Comment by cubells** [comment]
@pedrobaeza done. Wating to :green_heart:

**Pull request merged**